helm-charts/common/data-prep/values.yaml

```diff
@@ -5,17 +5,39 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-tei:
-  enabled: false
-milvus:
-  enabled: false
-redis-vector-db:
-  enabled: false
+# Configurations for OPEA microservice data-prep
+# Set it as a non-null string, such as true, if you want to enable logging.
+LOGFLAG: ""
+
+# data-prep needs to talk to different vector-DB backend, i.e. redis, milvus
+# Default is to use Redis as vector-DB backend
+DATAPREP_BACKEND: "REDIS"
+REDIS_HOST: ""
+REDIS_PORT: 6379
+REDIS_SSL: false
+REDIS_PASSWORD: ""
+REDIS_USERNAME: ""
+INDEX_NAME: "rag_redis"
+KEY_INDEX_NAME: "file-keys"
+SEARCH_BATCH_SIZE: 10
+
+# Uncomment and set the following settings to use Milvus as vector-DB backend
+# DATAPREP_BACKEND: "MILVUS"
+MILVUS_HOST: ""
+MILVUS_PORT: 19530
+COLLECTION_NAME: "rag_milvus"
+
+# retriever-usvc can do embedding locally or talk to remote tei service to do embedding
+# The following embedding related settings are listed in precedence order
+# text embedding inference service URL, e.g. http://<service-name>:<port>
```

Comment:
It's unclear what "following ... settings"  refers to.

Reply:
done